kyuubi-server/src/main/scala/org/apache/kyuubi/server/api/v1/BatchesResource.scala

```diff
@@ -545,7 +545,7 @@ private[v1] class BatchesResource extends ApiRequestContext with Logging {
       resourceFileInputStream: InputStream,
       resourceFileName: String,
       formDataMultiPartOpt: Option[FormDataMultiPart]): Option[JPath] = {
-    val uploadFileFolderPath = batchResourceUploadFolderPath(batchId)
+    val uploadFileFolderPath = KyuubiApplicationManager.sessionUploadFolderPath(batchId)
```

Comment:
Hi @turboFei @pan3793 , as batch id is not the same as the session id, is it a misuse here, uploaded files are put along with `batch id`, but the cleanup method cleans up the path with `session id` ?

Reply:
batchId == batch sessionId

Reply:
OK. Thanks